datalake/src/test/java/com/sequenceiq/datalake/service/sdx/SdxServiceTests.java

```diff
@@ -0,0 +1,266 @@
+package com.sequenceiq.datalake.service.sdx;
```

Comment:
Why is this a separate file?
We already have SdxServiceTest vs. SdxServiceTests. 

You are testing the same class SdxService.